examples/simple-platform/structure.md

```diff
@@ -0,0 +1,47 @@
+# Platform structure
+
+Our example has two React applications: `app-account` and 
+`interbit-template`. These apps are served in the user's browser as any other 
+React app would be. Either there is a static bundle served in a production 
+environment, or they are served from their own webpack servers in development 
+mode. Each React app starts an Interbit node locally, in the browser. The 
+platform also runs an Interbit node each for the `platform-deploy` and 
+`web-auth-endpoint` hypervisors. 
+
+The `platform-deploy` node hosts the accounts and template chains. (See 
+[Chain Sponsorship](../../reference/interbit-middleware/chainSponsorship.adoc) 
+for more information on hosting chains.)  It is the blockmaster and thus is 
+responsible for adding new blocks to the application chains. Note that there 
+is no requirement for the accounts and template chains to be hosted on the 
+same node. They could be hosted on separate nodes and moreover each app's 
```

Comment:
Omit `moreover`. If you feel that the emphasis is required: `...nodes and, more importantly, each app's...`